src/main/java/wolox/training/models/Book.java

```diff
@@ -0,0 +1,119 @@
+package wolox.training.models;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.validation.constraints.NotNull;
+
+@Entity
+public class Book {
+
+  @Id
+  @GeneratedValue(strategy = GenerationType.AUTO)
```

Comment:
Just a minor thing to change is the `GenerationType` here. We can use sequence instead of AUTO, giving that the auto generation type from Postgres uses a global auto increment id for all the tables (I mean, a shared one). So, with `AUTO`: if you insert a User, it will be persisted with id = 1. If you insert a Book after that, it will be persisted with id = 2.

Reply:
What @leandrocostawolox said ;)

Reply:
Done :) Thanks for the feedback